### DISCUSSION TITLE
Please talk about foods

### DISCUSSION THREAD

**Discussion by e061u** [NAIST-SE/AttractingContributors#100] [General]
https://github.com/iexuxs-m/AttractingContributors-Foods

**Comment by zhangyongxin1212** [comment]
https://github.com/zhangyongxin1212/AttractingContributors-Game/pulls
added and plz help me

**Comment by nagata-atsuki** [comment]
added!
Please help me!
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music
